build.gradle.kts

```diff
@@ -252,6 +253,12 @@ project(":") {
     tasks.withType<Jar> {
         dependsOn(copyXmls)
     }
+
+    tasks.withType<RunIdeTask> {
+        jvmArgs("-Xmx768m") // Default value for IDEA installation
```

Comment:
Mind, that CLion by default uses custom heap and stack sizes. Something like `-Xss2m -Xms256m -Xmx2000m`